### DISCUSSION TITLE
restorepreview patch doesn't seem to work anymore?

### DISCUSSION THREAD

**Discussion by limaceous-bushwhacker** [jarun/nnn#1776] [Q&A] [answered]
Title.

EDIT: nnn v4.9 with kitty 0.30.1

**Comment by luukvbaal** [comment, marked as answer]
Please show what you have tried/what exactly isn't working. Works fine on my end in `preview-tui` with `make O_RESTOREPREVIEW=1`.

**Reply by limaceous-bushwhacker** [reply]
Strange, it's not working here. See: 
![nnn](https://github.com/jarun/nnn/assets/67668779/33b400d8-c78a-463f-b8f9-d29b69a5820e)

Update: Fixed on updating preview-tui
